app/controllers/admin/themes_controller.rb

```diff
@@ -176,6 +176,7 @@ def index
       themes: serialize_data(@themes, ThemeSerializer),
       extras: {
         color_schemes: serialize_data(@color_schemes, ColorSchemeSerializer),
+        locale: I18n.locale,
```

Comment:
Maybe use `effective_locale` from the `User` model? Seems like the right thing, although `I18n.locale` seems to be doing the job too.

Reply:
Seems more correct indeed.

I'll change it